presto-hive/src/main/java/com/facebook/presto/hive/HiveOutputTableHandle.java

```diff
@@ -51,7 +53,8 @@ public HiveOutputTableHandle(
                 tableName,
                 inputColumns,
                 filePrefix,
-                requireNonNull(locationHandle, "locationHandle is null"),
+                locationHandle,
```

Comment:
This null check shouldn't have been added in the first place because the other arguments here don't have null test. And super will do the null test.